File: examples/python/clarify_image.py
from clarification import Options, clarify_file, sharpness_score

input_path = "input.png"
output_path = "clarified.png"

before = sharpness_score(input_path)
clarify_file(input_path, output_path, Options(amount=1.5, contrast=10.0))
after = sharpness_score(output_path)
print(f"sharpness: {before:.4f} -> {after:.4f}")

File: examples/lua/clarify_image.lua
local clarification = require("clarification")

local options = clarification.defaults()
options.amount = 1.5
options.contrast = 10.0

local output, err = clarification.enhance("clarification", "input.png", "clarified.png", options)
assert(output, err)
print("saved clarified.png")
